project/Build.scala

```diff
@@ -661,7 +661,8 @@ object Build {
           // running the compiler, we should always have the bootstrapped
           // library on the compiler classpath since the non-bootstrapped one
           // may not be binary-compatible.
-          "dotty-library"       -> packageBin.in(`dotty-library-bootstrapped`, Compile).value
+          "dotty-library"       -> packageBin.in(`dotty-library-bootstrapped`, Compile).value,
+          "dotty-staging"       -> packageBin.in(LocalProject("dotty-staging"), Compile).value,
```

Comment:
dotty-staging should never be used with a non-bootstrapped compiler so I don't think this line is needed.

Reply:
removed it but had to adapt some code in the test infrastructure